implementation/src/main/java/io/smallrye/metrics/MetricsRequestHandler.java

```diff
@@ -264,6 +264,10 @@ Optional<String> getBestMatchingMediaType(Stream<String> acceptHeaders) {
             }
         });
 
+        if (tupleList.isEmpty()) {
```

Comment:
Wouldn't it be simpler and more straightforward to change the condition on line 196  (`if (acceptHeaders == null)`) to also accept `acceptHeaders.count() == 0`?

Reply:
Works for me, I thought that method was the best to decide about the best matching media type and it is also easier to test.

Up to you ...

Reply:
Hmmm, ok I didn't realize it's more suitable for testing. Let's keep it this way then, thanks!